notNeededPackages.json

```diff
@@ -127,6 +127,12 @@
             "asOfVersion": "3.4.0"
         },
         {
+            "libraryName": "electron",
+            "typingsPackageName": "electron",
+            "sourceRepoURL": "https://github.com/electron/electron",
+            "asOfVersion": "1.7.2"
```

Comment:
Done.
But I still don't know what `asOfVesion` is, @andy-ms reviewed and replied that it should be 1.7.2 above